test/e2e/features/connections/connections-db.test.ts

```diff
@@ -14,7 +14,7 @@ test.describe('SQLite DB Connection', {
 	tag: [tags.WEB, tags.WIN, tags.CRITICAL, tags.CONNECTIONS]
 }, () => {
 	test.beforeAll(async function ({ userSettings }) {
-		await userSettings.set([['positron.connections.showConnectionPane', 'true']], true);
```

Comment:
did you confirm and/or deny that restart wasn't really needed?

Reply:
yeah, confirmed its not needed.  the new CI failure is a bug with the functionality I was adding a test for so I will add the bug reference and skip it for now.